src/js/controller/controller.js

```diff
@@ -386,14 +386,23 @@ Object.assign(Controller.prototype, {
             }
         }
 
-        function _checkAutoStart() {
+        const _checkAutoStart = () => {
             if (!apiQueue) {
                 // this player has been destroyed
                 return;
             }
 
+            const searchParams = new URLSearchParams(window.location.search);
+
+            // autostart playback at a specific point if we have a jw_start query
+            // parameter.
+            if (searchParams.has('jw_start')) {
+                // set autostart to viewable for analytics
+                this._model.setAutoStart('viewable');
+                _seek.call(this, parseInt(searchParams.get('jw_start')));
```

Comment:
This should probably be a `parseFloat` - nicer than including the "base" parameter to `parseInt` so that it doesn't interpret `023` as octal.